### PR TITLE
🎨 Email redesign #623

### LINKED ISSUE #623: :art:  Email Block Redesign: block redesign
## User stories

As a user, I want the "Email Block" component in the app to be redesigned to enhance its usability and visual appeal.

---

Quick links: [Summary](#summary) | [Description](#description) | [Template](#template) | [Example](#example) | [Resources](#resources)

---

### Technical Description
The current design is: 
image

It should be changed to:
image

The files can be found at `libs/features/convs-mgr/stories/blocks/library/email-message-block/`. 

The designs are [here](https://www.figma.com/file/ZGHzSstXOYPMFU1EiqjEHD/Untitled?type=design&node-id=1%3A4&mode=design&t=Hf7DxaUUMQ3BWy57-1)

### Acceptance Criteria

1. The block should look exactly like the designs.
2. Use of `rem`, `em` as opposed to `px`.
3. All changes should be isolated to the following  `libs/features/convs-mgr/stories/blocks/library/email-message-block/`.

### PR DESCRIPTION
**Pull request by dennis-mburu**
# Description
This PR redesigns the email block to enhance its usability and visual appeal

Fixes #623

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
email-menu-open
email-redesign-updated

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings

### CODE REVIEW

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/email-block.jsplumb.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ export function _EmailBlockDecoratePlumb(block: EmailMessageBlock, comp: Compone
     target: true,
     cssClass:"block_endpoint",
     endpoint: "Dot",
-    anchor: "Left",
+    anchor: [0 ,0.23, 0, 1],
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/email-block.jsplumb.ts:25]: Let's leave this as Left for now

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Kindly sync and resolve conflicts

**Comment by dennis-mburu** [comment]
I have updated accordingly. Kindly review @LemmyMwaura

**Review by LemmyMwaura — APPROVED** [review]
Add the new icon to the

**Pull request merged**